lib/acts_as_list/active_record/acts/list.rb

```diff
@@ -171,19 +171,12 @@ def self.move_after_record_with_position(records, targeted_record_position)
       module InstanceMethods
         # https://github.com/brendon/acts_as_list/commit/75ec5b9b3a7e72a35bc5bd95ca9061d44ece2a76
         def position_before_save_changed?
-          if ActiveRecord::VERSION::MAJOR >= 5 && ActiveRecord::VERSION::MINOR >= 1
```

Comment:
This conditional doesn't work very well if the version number is `x.0`, like `7.0`. I'd say it's safe to drop support for pre-Rails 5.1 code at this point though.